lib/configs/react.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
-  env: {
-    browser: true
+  parserOptions: {
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
```

Comment:
I _believe_ this is the configuration we want for React linting based on [language options guide](https://eslint.org/docs/latest/user-guide/configuring/language-options) and the config in [primer-react/recommended](https://github.com/primer/eslint-plugin-primer-react/blob/main/src/configs/recommended.js).